README.md

```diff
@@ -50,6 +52,21 @@ Job:new({
 
 A Lua module that implements a bunch of the things from `pathlib` from Python, so that paths are easy to work with.
 
+### plenary.scandir
+
+A Lua module for fast recursive file operations. Offers a recursive scan over a directory, similar to unix `find` or `fd`.
+It should be faster than running unix `find` running in a `plenary.job` and as fast as `fd` in a `plenary.job`.
```

Comment:
```suggestion
`plenery.scandir` is fast recursive file operations. It is similar to unix `find` or `fd` in that it can do recursive scan over a given directory, or a set of directories. 
```

Reply:
I like the part where we are faster than find and as fast as fd but thanks :)